dev-client/src/translations/en.json

```diff
@@ -150,7 +150,8 @@
       "delete": "Delete this project",
       "delete_button": "Delete",
       "delete_button_prompt": "Delete Project?",
-      "delete_description": "This will permanently delete the project’s data, as well as all of its sites and site data. This cannot be undone.\n\nAlternatively, project managers can transfer the sites out of this project to retain their data. "
+      "delete_description": "This will permanently delete the project’s data, as well as all of its sites and site data. This cannot be undone.\n\nAlternatively, project managers can transfer the sites out of this project to retain their data. ",
```

Comment:
```suggestion
      "delete_description": "This will permanently delete the project’s data, as well as all of its sites and site data. This cannot be undone.\n\nAlternatively, project managers can transfer the sites out of this project to retain their data.",
```

Reply:
suggestion: use <Trans> to break into paragraphs.

Reply:
Good idea with the `<Trans>`, I'll take another look at that later.